pyuvdata/uvcal.py

```diff
@@ -555,7 +555,7 @@ def read_calfits(self, filename, run_check=True, check_extra=True,
         Read in data from a calfits file.
 
         Args:
-            filename: The calfits file to read to.
+            filename: The calfits file or list of files to read from.
```

Comment:
Put the allowed types for each, just to make super clear. 
"filename=string path, or filename list or tuple of string paths."